web/viewer.js

```diff
@@ -203,6 +203,16 @@ function webViewerLoad() {
 
     window.PDFViewerApplication = pdfjsWebApp.PDFViewerApplication;
     window.PDFViewerApplicationOptions = pdfjsWebAppOptions.AppOptions;
+
+    if (typeof PDFJSDev !== 'undefined' && PDFJSDev.test('GENERIC')) {
+      // Give custom implementations of the default viewer a simpler way to
+      // set various `AppOptions`, by dispatching an event once all viewer
+      // files are loaded but *before* the viewer initialization has run.
+      const event = document.createEvent('CustomEvent');
+      event.initCustomEvent('webviewerload', true, true, {});
```

Comment:
Would it be better to call this `webviewerloaded` to match https://github.com/mozilla/pdf.js/blob/master/web/app.js#L870?

Reply:
That's a really good point about consistency; however considering that the entire viewer isn't actually loaded at this point, that name seems ever so slightly "off" to me.
How about `webviewerinit` instead, since there's already a bunch of similarly named events (e.g. documentinit, baseviewerinit, pagesinit)?